pkg/ebpf/events_pipeline.go

```diff
@@ -644,6 +653,55 @@ func (t *Tracee) getStackAddresses(stackID uint32) []uint64 {
 	return stackAddresses[0:stackCounter]
 }
 
+// enrichProcess is a pipeline stage which enrich the event with processes information.
+// This is done using the process tree of Tracee.
+func (t *Tracee) enrichProcess(ctx context.Context, in <-chan *trace.Event) (
+	<-chan *trace.Event, <-chan error,
+) {
+	out := make(chan *trace.Event)
+	errc := make(chan error, 1)
+
+	go func() {
+		defer close(out)
+		defer close(errc)
+
+		for {
+			select {
+			case event := <-in:
+				currentProcess, found := t.processTree.GetProcessByHash(event.ProcessEntityId)
+				if !found {
+					logger.Debugw(
+						fmt.Sprintf(
+							"error enriching event of pid %d tid %d with process info",
+							event.HostProcessID,
+							event.HostThreadID,
+						), "error", "not found",
+					)
+				} else {
+					executable := currentProcess.GetExecutable()
+
+					if executable != nil {
+						event.Executable = trace.Executable{
+							ExecutionTime: 0, // TODO: Get exec time
```

Comment:
Execution time is a property of the process and not of the executable. Let's omit it for now (anyway it is marked todo here)